README.md

```diff
@@ -160,9 +160,9 @@ This tells the style dictionary build system how and what to build. The default
 }
 ```
 
-Here we are creating some basic font size properties. The style definition size.font.small.value is "10px" for example.  The style definition size.font.base.value is automatically aliased to the value found in size.font.medium.value, so both of those resolve to "16px".
+Here we are creating some basic font size properties. The style definition size.font.small.value is "10px" for example. The style definition size.font.base.value is automatically aliased to the value found in size.font.medium.value and both of those resolve to "16px".
 
-Now what the style dictionary build system will do with this information is convert it to different formats so that you can use these values in any type of codebase. From this one file you can generate any number of files like:
+Now what the style dictionary build system will do with this information is convert it to different formats by which you can use these values in any type of codebase. From this one file you can generate any number of files like:
```

Comment:
Now what the style dictionary build system will do with this information is convert it to different formats, enabling these values ti be used in any type of codebase. From this one file you can generate any number of files like:

Reply:
"values to"